checks/evaluation/pinned_dependencies_test.go

```diff
@@ -303,40 +311,47 @@ func Test_PinningDependencies(t *testing.T) {
 					Location: &checker.File{
 						Snippet: "actions/checkout@v2",
 					},
-					Type:   checker.DependencyUseTypeGHAction,
-					Pinned: asBoolPointer(false),
+					Type:        checker.DependencyUseTypeGHAction,
+					Pinned:      asBoolPointer(false),
+					Remediation: testRemediation,
```

Comment:
was there a reason to add all these Remediation fields? Any extra test parts adds confusion on if its needed or not for the test

Reply:
need to update to get this merged anyway, so will remove + rebase/merge commit

Reply:
Done in https://github.com/ossf/scorecard/pull/3667/commits/4d7886c59d6efe089dc69886ba4fe05ae2d6af65